greenlet.c

```diff
@@ -153,6 +153,21 @@ static PyObject* ts_empty_dict;
 #define GREENLET_tp_is_gc 0
 #endif /* !GREENLET_USE_GC */
 
+static void green_clear_exc(PyGreenlet *g)
```

Comment:
Could you please change it to `PyGreenlet* g`?

Reply:
sure, sorry, fixed